src/backend/githubPlugin.test.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+
+import {GithubParser, getNodeType, getEdgeType} from "./githubPlugin";
+import exampleRepoData from "./githubDemoData/example-repo.json";
+
+describe("GithubParser", () => {
+  describe("whole repo parsing", () => {
+    const parser = new GithubParser("sourcecred/example-repo");
+    parser.addData(exampleRepoData.data);
+    const graph = parser.graph;
+
+    it("parses the entire example-repo as expected", () => {
+      expect(graph).toMatchSnapshot();
+    });
+
+    it("every comment has an author and container", () => {
+      const comments = graph
+        .getAllNodes()
+        .filter((n) => getNodeType(n) === "COMMENT");
+      comments.forEach((c) => {
```

Comment:
Let's assert that there are not zero comments (and likewise for issues and PRs).